Sources/Application/Environment/AppEnvironment.swift

```diff
@@ -174,6 +174,49 @@ extension AppEnvironment {
         string(for: .externalBaseURL)
     }
     
+    static var govURLString: String {
+        string(for: .govURLString)
+    }
+}
+
+// MARK: - Settings Page URLs
+    
+extension AppEnvironment {
+    
+    static var manageAccountURL: URL {
+        var components = URLComponents()
+        components.scheme = "https"
+        components.host = govURLString
+        components.path = "/using-your-gov-uk-one-login"
+        return components.url!
+    }
+    
+    static var manageAccountURLWelsh: URL {
+        var components = URLComponents()
+        components.scheme = "https"
+        components.host = govURLString
+        components.path = "/defnyddio-eich-gov-uk-one-login"
+        return components.url!
+    }
```

Comment:
Think these could probably be combined as such which will make the call sight simpler 🙂 
```suggestion
    static var manageAccountURL: URL {
        var components = URLComponents()
        components.scheme = "https"
        components.host = govURLString
        components.path = isLocaleWelsh ? "/defnyddio-eich-gov-uk-one-login" : "/using-your-gov-uk-one-login"
        return components.url!
    }
```